conf/category-list.json

```diff
@@ -8,6 +8,13 @@
         { "name": "Stylistic Issues", "description": "These rules relate to style guidelines, and are therefore quite subjective:" },
         { "name": "ECMAScript 6", "description": "These rules relate to ES6, also known as ES2015:" }
     ],
+    "deprecated": {
+        "name": "Deprecated",
+        "description": "These rules have been deprecated and replaced by newer rules.",
+        "rules": [
+            { "deprecated": "no-native-reassign", "replacedBy": ["no-global-assign"] }
```

Comment:
Can this be calculated instead? The fewer files we have to update to deprecate something, the fewer mistakes we will make.